r2/r2/models/subreddit.py

```diff
@@ -536,12 +536,46 @@ def add_moderator(self, user, **kwargs):
 
         return super(Subreddit, self).add_moderator(user, **kwargs)
 
+    def _handle_mod_succession(self):
+        if len(self.moderators) == 0:
+            return
```

Comment:
This check isn't actually needed, as the for loop would just do nothing instead...but from something I asked a month or so ago on stackoverflow, iterating over an empty list would still take up some processing power and an extremely small amount of time, so I added it anyway.